packages/example/docker-compose.yml

```diff
@@ -17,5 +17,6 @@ services:
     build: .
     volumes:
       - ../../:/app
+    restart: always
```

Comment:
@adelsz I have just noticed that for me this restarts these containers when docker restarts - apparently even after a restart. I could have sworn I've used this before without that effect. In any case I think `on-failure` would do the job here - the aim was just fro the example container to restart until postgres was ready. There are other ways also but I _thought_ this would be simpler

Reply:
Agree, `on-failure` should be a better fit here. I should probably add a reconnect feature to the CLI at some point.